src/SFA.DAS.ApplyService.Application/Apply/Validation/DateNotInFutureValidator.cs

```diff
@@ -6,14 +6,13 @@
 
 namespace SFA.DAS.ApplyService.Application.Apply.Validation
 {
-    public class DateNotInFutureValidator : IValidator
+    public class DateNotInFutureValidator : Validator
```

Comment:
As you're inheriting from an abstract class you really ought to be making these things a sealed class.
Not a biggie but worth keeping in mind

Reply:
I don't think this is a thing unless a article can be found to support sealing inherited abstract classes - I can't find anything on Google.